eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -320,16 +325,10 @@ jobs:
         # jacoco and spotbugs which are unnecessary for the From Source run.
         displayName: 'Generate Aggregate Pom for From Source Build'
         inputs:
-          pwsh: true
-          workingDirectory: $(Agent.BuildDirectory)
-          filePath: eng/scripts/Generate-Client-Aggregate-Pom.ps1
```

Comment:
Can this script be deleted now? - [Generate-Client-Aggregate-Pom](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/scripts/Generate-Client-Aggregate-Pom.ps1)

Reply:
Most likely, I'd leave it around for a while (month or so) while this change settles and is validated completely